.pre-commit-config.yaml

```diff
@@ -12,7 +12,7 @@ repos:
     rev: v4.0.1
     hooks:
       - id: check-added-large-files # Don't accidentally commit giant files.
-        args: ["--maxkb=10000"]
+        args: ["--maxkb=100000"]
```

Comment:
What do you think the cutoff should be for git vs GCS?

Reply:
GitHub's max file size is 100 MB, and Git LFS is 2 GB. I think it's simplest not to use the GCS bucket.

I don't think I was following this policy when I added the census zipfiles to the bucket. I could delete the GCS bucket in favor of Git LFS but it could be nice to keep around if we ever need a dataset that is larger that 2GB (aka PUDL).